app/routes/_mainLayout.t8_.$character.nav.tsx

```diff
@@ -0,0 +1,22 @@
+import { NavLink } from '@remix-run/react'
```

Comment:
When creating a component in the routes folder, it gets a separate url. So this will render if you go to the url /t8/feng/nav.
Can you move this file from app/routes to app/components? And maybe rename the file to just nav.tsx?